test/performance/dataplane-probe/main.go

```diff
@@ -30,11 +31,17 @@ import (
 
 var (
 	target = flag.String("target", "", "The target to attack.")
+	tags   = flag.String("tags", "", "The additional tags for this run in `a,b...,z` format")
```

Comment:
If we put this into `config-mako` then I can change this on my cluster (similar to env) without having to touch the benchmark code itself or risk accidentally checking in a change here.  I'd put it there and add helpers like `mako.GetBenchmark()`

Reply:
Isn't that proto owned by mako team?

Reply:
I'm talking about this: https://github.com/knative/serving/blob/4dea67050e45f17d48d9753e2d3b124eb8d5c218/test/performance/config/config-mako.yaml#L42

Reply:
Add it here: https://github.com/knative/serving/blob/4dea67050e45f17d48d9753e2d3b124eb8d5c218/test/performance/mako/config.go#L31